yjit/src/core.rs

```diff
@@ -311,6 +320,9 @@ pub struct Context {
 
     // Mapping of temp stack entries to types we track
     temp_mapping: [TempMapping; MAX_TEMP_TYPES],
+
+    // A block that is supplied by the caller to this Context
+    block_iseq: Option<IseqPtr>,
```

Comment:
In another PR, it might be good to make `ISeqPtr` a `NonNull<rb_iseq_t>` so that the memsize of `Option<ISeqPtr>` is just equal to the raw pointer size